openwisp_utils/admin_theme/static/rest_framework/css/browsable-api.css

```diff
@@ -0,0 +1,14 @@
+.meta.nocode {
+    display: block;
+    margin-top: -15px;
+    margin-bottom: 10px;
```

Comment:
`white-space: nowrap;` is needed here otherwise when there's no minification it will look as follows:

![Screenshot from 2021-05-25 20-21-22](https://user-images.githubusercontent.com/841044/119588621-cf900b80-bd96-11eb-9b12-a57e2d3926f1.png)

I fixed it in the latest commit.